roles/openshift_logging/defaults/main.yml

```diff
@@ -11,6 +11,8 @@ openshift_logging_install_logging: False
 openshift_logging_purge_logging: False
 openshift_logging_image_pull_secret: ""
 
+openshift_logging_image: "{{ l_os_registry_url | regex_replace(l_openshift_logging_search | regex_escape, 'logging-deployer') }}"
```

Comment:
where does 'logging-deployer' come from?  We have not utilized a deployer image in 4+ releases

Reply:
roles/openshift_logging/templates/jks_pod.j2

I'm totally unaware of what is current state in most of these roles, just trying to get the images uniform.  If we don't need that any more, we can get another PR to cut it out.